sqlglot/transforms.py

```diff
@@ -41,6 +41,18 @@ def unalias_group(expression: exp.Expression) -> exp.Expression:
 
     return expression
 
+def replace_db_to_schema( expression: exp.Expression) ->exp.Expression:
```

Comment:
Why not add this method into nuodb dialect specifically? rather than generic dialect, as so far, only nuodb needs to transpile a db into a schema. Unless other dbs do the same